src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/OdsRepository.cs

```diff
@@ -21,12 +21,12 @@ public OdsRepository(OdsSettings settings)
         public async Task<OdsOrganisation> GetBuyerOrganisationByOdsCodeAsync(string odsCode)
         {
             var response = await _settings.ApiBaseUrl
-                .AppendPathSegment("organisations")
+                .AppendPathSegments("ORD/2-0-0/organisations")
```

Comment:
Could you explain why you decided to move the API version out of config?

Reply:
As I was doing the integration tests I came to question my decision of including this in the 'baseUrl' because of my association of baseUrls to bare urls of the servers.

Would you say there's any advantage of keeping it the way it used to be?

Reply:
The only real advantage that I can think of would be in the event of a minor (non-breaking) change to the ODS (say a bug fix or minor update that takes it to 2-0-1 or 2-1-0).

If we wanted to make use of the updated version we'd currently have to make a code change.  However, with the version in config it would just involve updating the setting/environment variable.

Admittedly, this may be something of an edge case though.

Reply:
I can see your point. and because the change is not that big, I'll revert to having the version included in the base url.